content/installation/java/installation/AWS.md

```diff
@@ -0,0 +1,47 @@
+<!--
+title: "Installing on AWS"
+description: "AWS installation process"
+tags: "java agent installation AWS ebs elasticbeanstalk"
+-->
+
+Amazon Web Services (AWS) is a secure cloud services platform that provides a reliable, scalable and affordable computing platform solution. By using AWS, Contrast users can maximize the security of their applications.
+
+Contrast works with the following AWS systems:
+
+* AWS Elastic Beanstalk
+* Amazon EC2
+* Amazon RDS
```

Comment:
@dhafley: I feel like by including RDS specifically, we imply that is the only supported database.

Reply:
It is all relational databases which RDS serves in my opinion